src/runtime/recipe/ttl.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {assert} from '../../platform/assert-web.js';
+
+export enum TtlUnits {Minute = 'm', Hour = 'h', Day = 'd'}
```

Comment:
Have we considered supporting any of: months, years, seconds?

Reply:
seconds is too granular, i don't think we can guarantee deletion within seconds. 
months and years are trickier - because different months can have different amount of days, and there are leap year, so I just went with the simplest set of options. One can have 90 days or 1095 days ttl for 3 months or years accordingly. Or we can extend the enum, if needed.

Reply:
Sounds good.